docs/jwst/srctype/description.rst

```diff
@@ -5,19 +5,23 @@ determine whether a spectroscopic source should be considered to be a point
 or extended object, populating the "SRCTYPE" keyword with a value of either
 "POINT" or "EXTENDED."
 This information is then used in some subsequent spectroscopic processing
-steps.
+steps to apply source-dependent corrections.
 
-Depending on the JWST observing mode, the observer may have the option of
-designating a source type in the APT template for the observation. They have
-the choice of declaring whether or not the source should be considered
-extended. If they don't know the character of the source, they can also
-choose a value of "UNKNOWN." The observer's choice in the APT is passed along
+Single Source Observations
+--------------------------
+For JWST observing modes that use a single primary target (e.g. MIRI MRS
+and LRS spectroscopy and NIRSpec IFU and Fixed-Slit spectroscopy), the observer
+has the option of designating a source type in the APT template for the
+observation. They have the choice of declaring whether or not the source
+should be considered extended. If they don't know the character of the source,
+they can also choose a value of "UNKNOWN." The observer's choice is passed along
 to DMS processing, which sets the value of the "SRCTYPE" keyword in the
 primary header of the uncalibrated (_uncal.fits) product that's used as input
 to the calibration pipeline. If the user has selected a value in the APT, the
 "SRCTYPE" keyword will be set to "POINT", "EXTENDED", or "UNKNOWN." If the
```

Comment:
Does this mean that SRCTYPE will be set to the value the use selected?

Reply:
Yes. I could perhaps change the wording to make that more explicit.